README.md

```diff
@@ -81,6 +81,20 @@ GOOS=linux GOARCH=arm64 go build -ldflags "-s -w" -o xray cmd/tracing/daemon.go
 ```
 As of Aug 31, 2020, windows and darwin builds for ARM64 are not supported by `go build`.
 
+## Pulling X-Ray Daemon image from ECR Public Gallery
+####  Pull alpha tag from Public ECR Gallery
+``
+docker pull public.ecr.aws/xray-daemon/amazon/aws-xray-daemon:alpha
```

Comment:
Is this the final URL? It's better than the random string but we'll have `public.ecr.aws/amazon/aws-xray-daemon` right? I don't think we should update README until thats done to reduce confusion

Reply:
it's not final url but it will look something like this. ECR has this format `public.ecr.aws/<alias>/<repository-name>` so in our case our repository name is `amazon/aws-xray-daemon` (https://gallery.ecr.aws/e2w6v5y3/amazon/aws-xray-daemon). We can request custom alias to ECR team and that can be changed. If we request `xray-daemon` then url will look like something mentioned in the README. I am not going in to merge this PR until we have final confirmation on names. I have just staged so when we have approval on alias we can move forward quickly.

Reply:
Cool - I want a URL with no alias :) Even our ot collector doesn't have it - https://gallery.ecr.aws/aws-observability/aws-otel-collector

If we need to change the namespace from `amazon` to `awsxray` to have a name without alias, that would be better.

Reply:
In the case of collector repository name is `aws-otel-collector` and alias is `aws-observability`.  We can't get away with alias as per my understanding.